pkg/debug/info.go

```diff
@@ -0,0 +1,140 @@
+package debug
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/golang/glog"
+	"github.com/openshift/odo/pkg/occlient"
+	"github.com/openshift/odo/pkg/testingutil/filesystem"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+type OdoDebugFile struct {
+	metav1.TypeMeta
+	DebugProcessId int
+	ProjectName    string
+	AppName        string
+	ComponentName  string
+	RemotePort     int
+	LocalPort      int
+}
+
+// GetDebugInfoFilePath gets the file path of the debug info file
+func GetDebugInfoFilePath(client *occlient.Client, componentName, appName string) string {
+	tempDir := os.TempDir()
+	debugFilePrefix := "odo-debug.json"
```

Comment:
why is this called `debugFilePrefix`? shouldn't it be called `Suffix` as it comes at the end?

Reply:
fixed